pkg/convert/pot.go

```diff
@@ -73,46 +53,175 @@ func FromPot(b []byte) (model.Messages, error) {
 	}
 
 	messages := make([]model.Message, 0, len(po.Messages))
-	// model.Messages does not support plurals atm. But we plan to impl it under:
-	// https://github.com/orgs/translate-agent/projects/1?pane=issue&itemId=21251425
+
 	for _, node := range po.Messages {
-		if strings.HasSuffix(node.MsgId, "\\n") {
-			node.MsgId = strings.ReplaceAll(node.MsgId, "\\n", "\n")
+		message := model.Message{ID: node.MsgId, PluralID: node.MsgIdPlural}
```

Comment:
Declare and initialize all values at the same time. As @VladislavsPerkanuks already mentioned.

```suggestion
		message := model.Message{
			ID: node.MsgId, 
			PluralID: node.MsgIdPlural
			Description: strings.Join(node.ExtractedComment, "\n "),
			Fuzzy: strings.Contains(node.Flag, "fuzzy"),
		}
```